extension/generate-config.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
I feel some headaches about this shell script. 
1. It introduces another script language on top of Java, golang, and typescript 
2. It introduces additional logic on top of the build systems maven, go and NPM.
3. Potentially has limited portability, e.g. on Windows OS or other linux distros.
4. It could cause some code duplication as the script probably it will look similar for other VS.

I am fine to leave it as it is for the current PR, but I'd love if we could find a better solution in general.